--- a/Makefile	2017-03-16 12:24:24.000000000 -0500
+++ b/Makefile	2017-08-10 20:19:19.000000000 -0500
@@ -17,7 +17,7 @@
 RM = rm -f
 
 # On some platforms you might need to comment this out:
-CFLAGS += -std=c90 
+CFLAGS += -std=c89 
 
 CFLAGS += -O3 -funroll-loops -fomit-frame-pointer
 
@@ -32,7 +32,7 @@
 # Enable high resolution timers:
 CPPFLAGS += -DPNGCRUSH_TIMERS=11 -DPNGCRUSH_USE_CLOCK_GETTIME=1
 # If you get a linking error with clock_gettime() you might need this:
-# LIBS += -lrt
+LIBS += -lrt
 
 # Cannot use this with libpng15 and later.
 # CPPFLAGS += -DINFLATE_ALLOW_INVALID_DISTANCE_TOOFAR_ARRR
